dojo/filters.py

```diff
@@ -1544,6 +1544,7 @@ def filter(self, qs, value):
 class FindingFilterHelper(FilterSet):
     title = CharFilter(lookup_expr="icontains")
     date = DateFromToRangeFilter(field_name='date', label="Date Discovered")
+    date_recent = DateRangeFilter(field_name="date", label="Relative Date")
```

Comment:
```suggestion
    date = DateRangeFilter(field_name="date", label="Date Discovered")
```
Reverting the `date` to being the `DateRangeFilter` will allow us to avoid having to update any dashboard tile changes, and potentially other places that reference the `date` field as an integer (for a choice in the drop down specified in the `DateRangeFilter` class). I cannot make the full suggestion in Github to remove the current `date` filter since it is not in the diff, but the following line needs to be removed:
```
date = DateFromToRangeFilter(field_name='date', label="Date Discovered")
```

The same functionality post removal of the above will be preserved due to the existence of the `on/before/after` filters